delta/service/src/main/resources/service.conf

```diff
@@ -3,7 +3,7 @@ service {
     "ch.epfl.bluebrain.nexus.delta.sdk.model.Event"                                                         = "circe"
     "java.nio.file.Path"                                                                                    = "kryo"
     "org.apache.jena.iri.IRI"                                                                               = "kryo"
-    "org.apache.jena.rdf.model.Model"                                                                       = "kryo"
+    "org.apache.jena.sparql.core.DatasetGraph"                                                              = "kryo"
```

Comment:
the DatasetGraph package seems a bit odd... is this capability strictly tied to sparql?

Reply:
That't the capability we want and apparently is tied to that.